tests/test_passes.py

```diff
@@ -733,6 +732,68 @@ def test_nested_elimination(self):
         pyrtl.working_block().sanity_check()
 
 
+class TestSynthPasses(NetWireNumTestCases):
+    in1: pyrtl.Input
+    in2: pyrtl.Input
+    out: pyrtl.Output
+
+    def setUp(self):
+        pyrtl.reset_working_block()
+        self.in1 = pyrtl.Input(bitwidth=5, name='in1')
+        self.in2 = pyrtl.Input(bitwidth=5, name='in2')
+        self.out = pyrtl.Output(bitwidth=5, name='out')
+
+    def run_nand_synth(self, values: dict[tuple[int, int], int]):
+        pyrtl.synthesize()
+        pyrtl.nand_synth()
+        sim_trace = pyrtl.SimulationTrace()
+        sim = pyrtl.Simulation(tracer=sim_trace)
+        for inputs, output in values.items():
+            sim.step({'in1': inputs[0], 'in2': inputs[1]})
```

Comment:
Consider renaming `in1` and `in2` to `in0` and `in1` so this line can be `sim.step({'in0': inputs[0], 'in1': inputs[1]})`, which is more consistent